Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-getCollectionName.txt

MongoDB\\Collection::getCollectionName()
========================================

.. meta::

>    :description: Retrieve the name of a MongoDB collection using the `getCollectionName()` method in PHP.

Reviewer: ```suggestion
   :description: Retrieve the name of a collection with the MongoDB PHP Library.
```
